kolibri/core/auth/models.py

```diff
@@ -1415,6 +1415,23 @@ def add_coaches(self, users):
     def remove_coach(self, user):
         self.remove_role(user, role_kinds.COACH)
 
+    @classmethod
+    def get_or_create(cls, facility_name):
```

Comment:
I am not sure what this class method is giving us over just using `Facility.objects.get_or_create(name=facility_name)`

Reply:
Oh just me not knowing or checking if that's a built-in function